src/lib/providers/CurrentDomainProvider.tsx

```diff
@@ -21,11 +24,11 @@ export const CurrentDomainContext = React.createContext({
 });
 
 const parseDomainFromURI = (pathname: string) => {
-	if (pathname.startsWith('/market')) {
+	if (pathname.startsWith(ROUTES.MARKET)) {
 		return (
-			pathname.replace('/market', '') === ''
+			pathname.replace(ROUTES.MARKET, '') === ''
 				? '/'
-				: pathname.replace('/market', '')
+				: pathname.replace(ROUTES.MARKET, '')
 		).substring(1);
 	}
```

Comment:
This function looks a lot like the `pathname to ZNA` one we have in the lib - could be good to reuse it.

Reply:
Nice thanks. Updated. 🚀